promql/test.go

```diff
@@ -43,9 +44,16 @@ import (
 var (
 	minNormal = math.Float64frombits(0x0010000000000000) // The smallest positive normal value of type float64.
 
-	patSpace       = regexp.MustCompile("[\t ]+")
-	patLoad        = regexp.MustCompile(`^load\s+(.+?)$`)
-	patEvalInstant = regexp.MustCompile(`^eval(?:_(fail|ordered))?\s+instant\s+(?:at\s+(.+?))?\s+(.+)$`)
+	patSpace                = regexp.MustCompile("[\t ]+")
+	patLoad                 = regexp.MustCompile(`^load\s+(.+?)$`)
+	patEvalInstant          = regexp.MustCompile(`^eval(?:_(fail|ordered))?\s+instant\s+(?:at\s+(.+?))?\s+(.+)$`)
+	histogramBucketReplacer = strings.NewReplacer(
+		"_bucket", "",
+		" by (le)", "",
+		"(le, ", "(",
+		", le, ", ", ",
+		", le)", ")",
```

Comment:
Nice, let's follow the pattern above where whitespace is matched with `\s`

```suggestion
		"\s+by\s*(\s*le\s*)", "",
		"(\s*le\s*,\s*", "(",
		",\s*le\s*,\s*", ", ",
		",\s*le\s*)", ")",
```